src/openrct2-ui/windows/ObjectLoadError.cpp

```diff
@@ -119,7 +119,7 @@ class ObjectDownloader
                 }
                 else
                 {
-                    throw std::runtime_error("Non 200 status");
+                    std::printf("  Failed to download %s\n", name.c_str());
```

Comment:
Is there a specific reason for the two spaces at the start of the string?

Reply:
This is a copy of the one below which I indented just so that it stood out from the "downloading object..." status messages.